operator-framework-core/src/main/java/io/javaoperatorsdk/operator/processing/event/source/informer/InformerEventSource.java

```diff
@@ -275,6 +295,9 @@ public boolean allowsNamespaceChanges() {
     return configuration().followControllerNamespaceChanges();
   }
 
+  public boolean parseResourceVersions() {
```

Comment:
This is not used.

Reply:
Right, it had been in some early code, but is no longer needed.